### DISCUSSION TITLE
[Icon Suggestion] Crown

### DISCUSSION THREAD

**Discussion by PolMrt** [tailwindlabs/heroicons#481] [Ideas]
![image](https://user-images.githubusercontent.com/28253868/108326387-74b0ac00-71ca-11eb-905b-96f1f338371f.png)

```html
<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="currentColor">
  <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M19.4142 20.4142C19.0391 20.7893 18.5304 21 18 21C17.4696 21 14.4153 19.5 12 19.5C9.58474 19.5 6.53043 21 6 21C5.46957 21 4.96086 20.7893 4.58579 20.4142L2 11L6 15V6L10 13L12 3L14 13L18 6V15L22 11L19.4142 20.4142Z" />
</svg>
```

**Comment by tgaeta** [comment]
Bump

**Comment by ndri** [comment]
I added `crown` to [Sidekickicons](https://github.com/ndri/sidekickicons/), which are meant to be used as a companion to Heroicons. Browse them here: https://sidekickicons.com/?icon=crown

**Reply by tgaeta** [reply]
@ndri ![you da real mvp](https://media1.tenor.com/m/YeLzZm4BFL8AAAAC/kevin-durant-real-mvp.gif)
